### PR TITLE
Add CI creates issue updating dependencies

### PR DESCRIPTION
**Pull request by VasiliySoldatkin**
Adds CI, which uses cron GHA to create Issue "Upgrade dependencies" every 3 months 
Context: [#2569](https://github.com/meilisearch/meilisearch/issues/2569)

### CODE REVIEW

File: .github/workflows/create-issue-dependencies.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Create Issue "Upgrade dependencies"
```
Comment (curquiza) [inline .github/workflows/create-issue-dependencies.yml:1]:
```suggestion
name: Create issue to upgrade dependencies
```

File: .github/workflows/create-issue-dependencies.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    - cron: '0 0 1 */3 *'
+  workflow_dispatch:
+  
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v1
```
Comment (curquiza) [inline .github/workflows/create-issue-dependencies.yml:13]:
```suggestion
      - uses: actions/checkout@v3
```

**Review by curquiza — CHANGES_REQUESTED** [review]
Hello @VasiliySoldatkin

Thanks for this PR! This is really helpful
I would rather we don't need to create a new template, otherwise it will be suggested when creating an issue, which is not something we want
I've done some research and I think we could you this GHA: https://github.com/actions-ecosystem/action-create-issue

Thanks again for your PR!! ❤️

**Comment by curquiza** [comment]
👆I've applied directly my small suggestion of change on the branch, so don't forget to `git pull` on your branch before doing any new changes 😄

**Comment by VasiliySoldatkin** [comment]
Thank you @curquiza for review! I made some changes in .yml, is that what you need?

**Pull request merged**